lib/ansible/modules/packaging/os/rhn_register.py

```diff
@@ -35,7 +35,7 @@
 notes:
     - In order to register a system, rhnreg_ks requires either a username and password, or an activationkey.
 requirements:
-    - rhnreg_ks
+    - rhnreg_ks, either libxml2 or lxml
```

Comment:
I believe this is a yaml array. The hyphen up front indicates an array element. I believe the new requirements should be on a new line.

https://github.com/ansible/ansible/blob/devel/examples/DOCUMENTATION.yml#L31